skbio/stats/tests/test_composition.py

```diff
@@ -269,6 +355,136 @@ def test_centralize(self):
                             np.array([[2, 2, 6],
                                       [4, 4, 2]]))
 
```

Comment:
Another couple of test cases (I'm pulling these from the anosim/permanova tests if you were wondering):

- test with groups of unequal size
- test that you get the same results when passing `grouping` that is not defined with 0's and 1's (e.g., `[0,0,0,1,1,1]` and `['foo','foo','foo','bar','bar','bar']`)